### DISCUSSION TITLE
Native System Menu Bar Integration?

### DISCUSSION THREAD

**Discussion by nu11ptr** [emilk/egui#3293] [Q&A]
This probably isn't technically an egui specific question, but more of an integration question. If I wanted to create an egui app, but use a native system menu bar (esp. important on macOS)... is there a way to do it and has anybody done it? Either with eframe or some other set of windowing crates? (maybe using `menubar` crate???) thx

Update: One idea would be to use [this](https://crates.io/crates/fltk-egui) crate and use FLTK for the window creation and system menus and egui for the inner canvas.

Update 2: Another probably better idea is to use `egui-tao` along with [muda](https://github.com/tauri-apps/muda/blob/dev/examples/tao.rs) crates

**Comment by razein97** [comment]
For future visitors.
A fix is available here.
https://github.com/emilk/egui/issues/3411#issuecomment-3398463201
